.github/workflows/python-app.yml

```
@@ -13,24 +13,65 @@ permissions:
+    - uses: actions/checkout@v4
+    
+    - name: Set up Node.js
+      uses: actions/setup-node@v4
+      with:
+        node-version: '18'
+        cache: 'npm'
+    
+    - name: Install dashboard dependencies
+      working-directory: ./dashboard
+      run: npm ci
+    
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)